test/PushNotificationTests/PushNotification-Test-Constants.h

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#pragma once
+
+inline const winrt::hstring c_rawNotificationPayload = L"<toast></toast>";
+static const std::chrono::seconds c_timeout = std::chrono::seconds(300);
```

Comment:
What's the reason for this var being static instead of inline?

Reply:
Should be inline. Changing.